openlibrary/plugins/openlibrary/js/carousel/Carousel.js

```diff
@@ -198,7 +201,7 @@ const Carousel = {
 
                 // Remove the current slides
                 slick.removeSlide(totalSlides, true, true);
-                slick.addSlide('<div class="carousel__item carousel__loading-end">Loading...</div>');
+                slick.addSlide('<div class="carousel__item carousel__loading-end">i18nValues["loading"]</div>');
```

Comment:
```suggestion
                slick.addSlide(`<div class="carousel__item carousel__loading-end">${i18nValues["loading"]}</div>`);
```
Use template literals for string interpolation.